src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -525,14 +524,13 @@ describe('Schemaform <SaveInProgressIntro>', () => {
         user={user}
         fetchInProgressForm={fetchInProgressForm}
         removeInProgressForm={removeInProgressForm}
-        renderSignInMessage={renderSpy}
         toggleLoginModal={toggleLoginModal}
         formConfig={formConfig}
-        headingLevel="h1"
+        headingLevel={1}
```

Comment:
and updating this to use a number. the component proptypes should probably be updated to make sure we only pass in a number between 1 and 6, like the AlertBox does https://github.com/department-of-veterans-affairs/component-library/blob/be4ee7dc32ccc2d55cf23c242a84f7c3902f9a49/src/components/AlertBox/AlertBox.jsx#L177-L184